packages/example/pages/index.tsx

```diff
@@ -16,7 +16,7 @@ interface Summary {
   address: string;
   wallet: string;
   celo: BigNumber;
-  balances: { symbol: string; value: BigNumber }[];
+  balances: { symbol: StableToken; value: BigNumber | string }[];
```

Comment:
maybe a nicer way here would be to have `symbol: StableToken, value?: BigNumber, error?: string`

Reply:
added a commit that changes the value to also be of type string, should it still be split out into `value?: string, error?: string`?

Reply:
left a comment on the not outdate thread. I still think it should be split